testnets/001_valetudo.csv

```diff
@@ -0,0 +1,26 @@
+(200000, penumbra,      penumbrav0t1na5duj20at9k3faq8cvqf3ykk0vkh326q2klp9phz5vp2ndy3mnqx9x28yzrdxyc0p9qv68j4zegs4augceu6nj9g9ylnmxu0eg7st3mshkhsqjy7248x2gszpsfj9)
```

Comment:
This is not actually a CSV formatted file, which makes the extension a little misleading. Should we either (a) drop the parentheses, making it actually CSV, or (b) change the file extension?